app/controllers/api/v1/merchants/search_controller.rb

```diff
@@ -0,0 +1,17 @@
+class Api::V1::Merchants::SearchController < ApiBaseController
+  respond_to :json
+
+  def index
+    respond_with Merchant.where(merchant_params)
```

Comment:
Again, might need a @merchants = Merchant.all ... we'll see further down the road